client/webpack-hot-middleware-client.js

```diff
@@ -52,6 +52,18 @@ export default () => {
     }
   }
 
+  const {
+    __NEXT_DATA__: {
+      assetPrefix
```

Comment:
As above comment, a default value  might help?

Reply:
Next takes care of this in: https://github.com/zeit/next.js/blob/fe1b3e89f1d6253726074ee21755422b0c664197/server/config.js#L11

Reply:
👍